compiler/rustc_hir_analysis/src/hir_ty_lowering/dyn_compatibility.rs

```diff
@@ -276,7 +271,7 @@ impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
                         self.dcx(),
                         i.bottom().1,
                         E0038,
-                        "the {} `{}` cannot be made into an object",
+                        "the {} `{}` is not dyn-compatible",
```

Comment:
Obviously a nit, but do we want to hyphenate this use or not?  @fmease, how does this fit with the system you've been using elsewhere?

We didn't hyphenate over in:

https://github.com/rust-lang/lang-team/issues/286

My grammatical intuition is that in most uses it should be unhyphenated, as with "object safety" or "forward compatible".

Reply:
Updated!